README.md

```diff
@@ -13,10 +13,14 @@ This is an on-going project, supported by Nuxeo.
 
 ### Nuxeo Platform - JS Client Dependencies
 
+For older versions < 3.0:
+
 | Nuxeo Platform | JS Client |
 | -------------- |-----------|
 | LTS 2015       | >= 1.x.x  |
-| FT 8.x         | >= 2.x.x  |
+| LTS 2016       | >= 2.x.x  |
+
+Since the 3.0 version, the JS Client works with Nuxeo version >= LTS 2015.
```

Comment:
Remove the information about old versions. State only that JS Client is compliant all Nuxeo versions as of LTS 2015.